Dockerfile

```diff
@@ -82,10 +82,10 @@ COPY requirements /tmp/requirements
 RUN --mount=type=cache,target=/root/.cache/pip \
     set -x \
     && pip --disable-pip-version-check \
-            install --no-deps \
-            -r /tmp/requirements/docs-dev.txt \
-            -r /tmp/requirements/docs-user.txt \
-            -r /tmp/requirements/docs-blog.txt \
+    install --no-deps \
+    -r /tmp/requirements/docs-dev.txt \
+    -r /tmp/requirements/docs-user.txt \
+    -r /tmp/requirements/docs-blog.txt \
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** can you explain why the leading whitespace was removed here? We have it here and in other spots to increase readability of multi-line commands.‏

Reply:
I think it is done by `prettier`, Let me fix them.